test/unit/composite.js

```diff
@@ -243,8 +243,7 @@ describe('composite', () => {
     sharp(fixtures.inputJpg)
       .resize(300, 300)
       .composite([{
-        input: Buffer.from('<svg width="200" height="200"><rect x="0" y="0" width="200" height="200" rx="50" ry="50"/></svg>'),
-        density: 96,
+        input: fixtures.destInCutout,
```

Comment:
It can be, but I'd rather leave it. I worked on the resvg support, but it's fairly limited in non-obvious ways (text mostly) so I'd rather keep it as explicit opt-in.